includes/Dashboard.php

```diff
@@ -28,12 +28,15 @@
 
 namespace Google\Web_Stories;
 
+use Google\Web_Stories\TRAITS\Assets;
```

Comment:
Can we use `Traits` instead of `TRAITS` everywhere?

Reply:
fixed.